src/util/auth/api.ts

```diff
@@ -46,6 +46,7 @@ export interface Notification {
     Body: string;
     Timestamp: Timestamp;
     Type: NotificationType;
+    Data: string;
```

Comment:
is this guaranteed to be provided by the backend? i worry about existing notifications

Reply:
Added the frontend check to account for that; it'll be undefined if not provided